python/llm/test/benchmark/arc-perf-test2.yaml

```diff
@@ -0,0 +1,14 @@
+# for the models which required the version of transformers 4.34.0
```

Comment:
rename to arc-perf-transformers-434.yaml?

Reply:
ok